Documentation/Metrics/arangodb_maintenance_action_done_total.yaml

```diff
@@ -1,11 +1,11 @@
-name: arangodb_maintenance_action_done_msec_total
+name: arangodb_maintenance_action_done_total
 help: |
-  Accumulated number of action queue time for Maintenance.
+  Counter of actions that are done and have been removed from the registry.
 unit: ms
 type: counter
 category: Maintenance
-complexity: advanced
+complexity: simple
 exposedBy:
   - dbserver
 description: |
-  Accumulated action runtime for Maintenance.
+  Counter of actions that are done and have been removed from the registry.
```

Comment:
I think this should be a bit more elaborate, in particular if the complexity is simple. We should spend a few sentences on explaining what the maintenance is, what the expected values are and what to do if the values get out of hand.